src/Authentication/Model/AuthenticationEventType.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Altinn.Platform.Authentication.Model
+{
+    /// <summary>
+    /// Enumeration for authentication event types
+    /// </summary>
+    public enum AuthenticationEventType
+    {
+        Authenticated,
```

Comment:
Enum should be declared with values and 0 should be NotAuthenticated. Now the default will be Authenticated